frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -315,3 +315,47 @@ Feature: Topology chart area
               And user opens the details page for "nodejs-2" by clicking on the title
               And user navigate back to Topology page
              Then user will see the the workload "nodejs-2" selected with sidebar open
+
+
+        @regression @odc-5947
+        Scenario: Create Service Binding option in nodes actions menu: T-06-TC27
+            Given user has installed Service Binding operator
+              And user is at developer perspective
+              And user is at Topology page chart view
+              And user has created a deployment workload "node-js1"
+             When user right clicks on workload "node-js1"
+              And user clicks on "Create Service Binding" option from context menu
+             Then user will see "Create Service Binding" modal
+              And user will see alert "No bindable services available"
+
+
+        @regression @manual @odc-5947
+        Scenario: Display of External Bindable resources: T-06-TC28
+            Given user has installed Service Binding operator
+              And user is at developer perspective
+              And user is at Topology page chart view
+              And user has created a deployment workload "node-js"
+            #Please refer to test case KM-01-TC01 for creating kafka connection
+              And user has created external bindable resource Kafka Connection "kafka-instance-ex"
+             When user drag and drop the connector to Kafka Connection
```

Comment:
when the user drags and drops connector on the bindable service to create the service binding the name field in the modal is pre-populated with the default name for the service binding which the user can edit
default name format : `<source-name>-<source-model-in-lowercase>-<target-name>-<target-model-in-lowercase>`

Reply:
Updated